python/mxnet/io.py

```diff
@@ -80,7 +80,22 @@ def get_list(shapes, types):
             return [DataDesc(x[0], x[1]) for x in shapes]
 
 class DataBatch(object):
-    """A data batch.
+    """Returns a batch of data.
```

Comment:
Since this is a class, you are not describing what is does but what it encapsulates.